live/js.go

```diff
@@ -0,0 +1,198 @@
+package live
+
+import (
+	"encoding/json"
+	"strings"
+)
+
+// DefaultTransitionDuration_ms is the default duration, in milliseconds, of live.JS transitions.
+const DefaultTransitionDuration_ms = int32(200)
```

Comment:
Can we instead do `DefaultTransitionDuration = 200 * time.Millisecond` and then call `.Milliseconds()` on it when needed?